model/file_info_test.go

```diff
@@ -24,109 +24,186 @@ func TestFileInfoIsValid(t *testing.T) {
 		Path:      "fake/path.png",
 	}
 
-	require.Nil(t, info.IsValid())
-
-	info.Id = ""
-	require.NotNil(t, info.IsValid(), "empty Id isn't valid")
-
-	info.Id = NewId()
-	info.CreateAt = 0
-	require.NotNil(t, info.IsValid(), "empty CreateAt isn't valid")
-
-	info.CreateAt = 1234
-	info.UpdateAt = 0
-	require.NotNil(t, info.IsValid(), "empty UpdateAt isn't valid")
-
-	info.UpdateAt = 1234
-	info.PostId = NewId()
-	require.Nil(t, info.IsValid())
-
-	info.Path = ""
-	require.NotNil(t, info.IsValid(), "empty Path isn't valid")
-
-	info.Path = "fake/path.png"
-	require.Nil(t, info.IsValid())
+	t.Run("Original File Info", func(t *testing.T) {
+		assert.Nil(t, info.IsValid())
+	})
+
+	t.Run("Empty ID is not valid", func(t *testing.T) {
+		info.Id = ""
+		assert.NotNil(t, info.IsValid(), "empty Id isn't valid")
+		info.Id = NewId()
+	})
+
+	t.Run("CreateAt 0 is not valid", func(t *testing.T) {
+		info.CreateAt = 0
+		assert.NotNil(t, info.IsValid(), "empty CreateAt isn't valid")
+		info.CreateAt = 1234
+	})
+
+	t.Run("UpdateAt 0 is not valid", func(t *testing.T) {
+		info.UpdateAt = 0
+		assert.NotNil(t, info.IsValid(), "empty UpdateAt isn't valid")
+		info.UpdateAt = 1234
+	})
+
+	t.Run("New Post ID is valid", func(t *testing.T) {
+		info.PostId = NewId()
+		assert.Nil(t, info.IsValid())
+	})
+
+	t.Run("Empty path is not valid", func(t *testing.T) {
+		info.Path = ""
+		assert.NotNil(t, info.IsValid(), "empty Path isn't valid")
+		info.Path = "fake/path.png"
+	})
+
+	t.Run("Original file is still valid", func(t *testing.T) {
+		assert.Nil(t, info.IsValid())
+	})
 }
 
 func TestFileInfoIsImage(t *testing.T) {
-	info := &FileInfo{MimeType: "image/png"}
-	assert.True(t, info.IsImage(), "file is an image")
-
-	info.MimeType = "text/plain"
-	assert.False(t, info.IsImage(), "file is not an image")
+	info := &FileInfo{}
+	t.Run("Mime PNG is an image", func(t *testing.T) {
```

Comment:
I would change this to something like this for more clarity
```suggestion
	t.Run("MimeType set to image/png is considered an image", func(t *testing.T) {
```

Reply:
Should be corrected at 6fa5062 along with the other comments.

Thanks!